.github/workflows/release.yml

```diff
@@ -50,6 +42,13 @@ jobs:
         with:
           path: target/${{matrix.build-profile}}
           key: ${{runner.os}}-${{matrix.network}}-${{matrix.arch}}-${{env.RELEASE_VERSION}}
+      - name: Install Rust Toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          default: true
+          profile: minimal
+          target: wasm32-unknown-unknown
+          toolchain: nightly
```

Comment:
set action chain to nightly, this will be overridden by rust-toolchain file when cargo runs

Reply:
We wont have to change it again